docker-compose.build.yml

```diff
@@ -18,7 +18,9 @@ services:
             args:
                 BUILD_DATE: ${BUILD_DATE:?err}
                 BUILD_NUMBER: ${BUILD_NUMBER:?err}
-                BUILD_ARCH: ${BUILD_ARCH:?err}
+                TARGETARCH: ${BUILD_ARCH:?err}
+                TARGETOS: linux
+                TARGETVARIANT: ${BUILD_VARIANT}
```

Comment:
I've updated the docker-compose definition so it should still work.

Reply:
`BUILD_VARIANT` is optional as it's only used for armhf (the arch would be `arm` and the variant would be `v7` for instance).